Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env groovy
 
-library identifier: 'apm@master',
+library identifier: 'apm@v1.0.2',
```

Comment:
Changes in this file don't seem related.

Reply:
Hi Jaime, well honestly I don't known how this came in, but for sure it seems useless for me too. I'll come back to "master" in my next push then.